warehouse/table_upload.go

```diff
@@ -61,6 +64,27 @@ func createTableUploads(uploadID int64, tableNames []string) (err error) {
 	return err
 }
 
+func createTableUploads(uploadID int64, tableNames []string) (err error) {
+	// we add table uploads to db in batches to avoid hitting postgres row insert limits
+	numOfBatches := len(tableNames) / createTableUploadsBatchSize
+	for batch := 0; batch < numOfBatches; batch++ {
```

Comment:
do we need to do two times - once in for loop and and in if statement. can't we do in one loop something like where we do initUpload in warehouse.go

Reply:
@dhawal-rudder something like
```
for i := 0; i < len(tableNames); i += createTableUploadsBatchSize {
	j := i + createTableUploadsBatchSize
	if j > len(tableNames) {
		j = len(tableNames)
	}
	createTableUploadsForBatch(uploadID, tableNames[i:j])
}